README.md

```diff
@@ -25,12 +25,14 @@ this:
     b2 cancel-large-file <fileId>
     b2 clear-account
     b2 create-bucket [--bucketInfo <json>] [--corsRules <json>] [--lifecycleRules <json>] <bucketName> [allPublic | allPrivate]
+    b2 create-key [--duration <validDurationSeconds>] [--bucket <bucketId>] [--prefix <namePrefix>]
     b2 delete-bucket <bucketName>
     b2 delete-file-version [<fileName>] <fileId>
+    b2 delete-key <applicationKeyId>
     b2 download-file-by-id [--noProgress] <fileId> <localFileName>
     b2 download-file-by-name [--noProgress] <bucketName> <fileName> <localFileName>
     b2 get-account-info
-    b2 get-bucket <bucketName>
+    b2 get-bucket [--showSize] <bucketName>
```

Comment:
Was this an update we missed?

Reply:
When I ran the tool this came out

Reply:
That's what I thought.  When those previous changes went in, the README didn't get updated.

Thanks for fixing it!